api-reference/10 UI Components/Component/3 Methods/beginUpdate().md

```diff
@@ -6,7 +6,7 @@ id: Component.beginUpdate()
 Prevents the UI component from refreshing until the [endUpdate()](/api-reference/10%20UI%20Components/Component/3%20Methods/endUpdate().md '{basewidgetpath}/Methods/#endUpdate') method is called.
```

Comment:
Looks like now the short description contradicts the updated passage.

Can we also update the short description to say that the method, for instance, *"Postpones renders that can negatively affect performance until the `endUpdate()` method is called."*

P.S. I thought about "*reduces the number of renders until the `endUpdate()` method is called*" for the short description at first, but that would mean that I could possibly call `beginUpdate` without `endUpdate` to enter a sort of "efficient mode" with reduced renders. This is not true - I still need `endUpdate` to ensure that all the necessary renders happen.

Reply:
Ok, thanks. I considered it!